Cargo.toml

```diff
@@ -21,6 +21,9 @@ include     = [
     "resources/sounds/*"
 ]
 
+[target."cfg(windows)"]
```

Comment:
is this required to build for multiple arch? could it have any undesired impact?

Reply:
This specific table is [explained](https://volks73.github.io/cargo-wix/cargo_wix/index.html#c-runtime) in detail on the `cargo-wix` documentation. I just generalized the header to shorten it based on project specs.